index.html

```diff
@@ -34,18 +35,52 @@ <h1>Flight List</h1>
 
         </table>
       </div>
+=======
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+>>>>>>> Stashed changes
+
+
+    <link rel="stylesheet" href="./src/style.css" />
+    <link rel="stylesheet" href="/dist/output.css">
 
-      <div class="right-side">
-        <div class="map-container">
-          <div id="map"></div>
-        </div>
 
+    <title>Plane Tracker 2.0</title>
+  </head>
 
+  <body class="sm:flex sm:flex-col sm:h-screen">
+    <div class="flex justify-center bg-tahiti text-white">
+      <img class="logo py-2 my-2" src="./src/assets/plane.svg" alt="" />
+      <h1 class="text-3xl font-bold px-4 py-2 mx-4 my-2">Flight Tracker</h1>
+    </div>
+    <main class="sm:flex-grow">
+      <div class="flex flex-col md:flex-row">
+        <div class="w-full md:w-1/3 lg:w-1/2 mx-auto md:mx-0">
+          <!-- Left side content with 30% width -->
+          <div id="loader">
+            <div class="spinner"></div>
+          </div>
+          <table id="flight-list" class="w-full cursor-pointer mt-8">
+            <tr class="h-16 bg-tahiti text-white ">
+              <th class="px-4">Flight Code</th>
+              <th class="px-4">Country</th>
+              <th class="px-4">Flight Speed</th>
+            </tr>
+          </table>
+        </div>
+        <div class="w-full md:w-2/3 lg:w-1/2">
+          <!-- Right side content with 70% width -->
+          <div class="map-container">
+            <div id="map"></div>
+          </div>
+        </div>
       </div>
     </main>
 
     <script type="module" src="./src/plane-track.ts"></script>
-</body>
+  </body>
 
 </html>
 <script src="https://unpkg.com/leaflet@1.9.3/dist/leaflet.js"
```

Comment:
You have leaflet installed as a dependency you don't need to link to its external packages.

Reply:
Thanks, resolved.